DM-2812: Adding change logs (#2545)
SoftwareAG/c8y-docs: content/change-logs/device-management/lwm2m-agent_10-20-394-0_updating-bulk-registration-templates.md

---
date:
title: Updated LWM2M bulk registration templates
product_area: Device management & connectivity
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:
  - value: tc-ggH2M4hf3
    label: lwm2m-agent

> ticket: DM-4048

Reviewer: ```suggestion
ticket: DM-2812
```

version: 10.20.394.0
---

> The wizard for bulk registering LWM2M devices to Cumulocity provides default bulk registration templates.

Reviewer 2: ```suggestion
The wizard for bulk registering LWM2M devices to {{< product-c8y-iot >}} provides default bulk registration templates.
```

> When bulk registering LWM2M devices to the LWM2M service, the wizard provides default bulk registration templates.

Reviewer 3: ```suggestion
The wizard for bulk registering LWM2M devices to Cumulocity provides default bulk registration templates.
```

> These bulk device registration templates have been updated to reflect new properties including X.509 security

Reviewer 3: ```suggestion
These templates have been updated to reflect new properties including the X.509 security
```
